### ISSUE TITLE
Improve accessibility of timestamp permalink in timeline

### ISSUE THREAD

**Issue by t3chguy** [element-hq/element-web#11514]
It should be just a timestamp without the link and also without conditions on when it shows. Permalink is available from the options context menu so doesn't need duplicating here.

**Comment by MadLittleMods** [comment]
I like the permalink being on the timestamp of every message and was super intuitive coming from Gitter. But it doesn't give a nice a Matrix.to link like it should.

**Comment by psrpinto** [comment]
Assuming this issue is about the timestamp that currently displays below the message sender's avatar, I personally find it useful that it navigates to (i.e. selects) the message, and that I can then copy the URL in the browser to share that message. 

> and also without conditions on when it shows. 

@t3chguy Could you clarify what you mean by this, and why removing the link would improve accessibility?

**Comment by t3chguy** [comment]
Closing as I have no recall for why this would be a big deal
